au4/src/au3wrap/internal/au3project.cpp

```diff
@@ -50,6 +51,17 @@ std::shared_ptr<Au3Project> Au3Project::create()
 
 bool Au3Project::load(const muse::io::path_t& filePath)
 {
+    TrackList& tracks = TrackList::Get(m_data->projectRef());
+    mTrackListSubstription = tracks.Subscribe([this](const TrackListEvent& event)
+    {
+        if (event.mType == TrackListEvent::ADDITION) {
+            const auto tempo = ProjectTimeSignature::Get(m_data->projectRef()).GetTempo();
+            if (const auto track = event.mpTrack.lock()) {
+                DoProjectTempoChange(*track, tempo);
```

Comment:
We already do it here https://github.com/audacity/audacity/blob/master/au4/src/trackedit/internal/au3/au3trackeditproject.cpp#L103